specificatie/domain.yaml

```diff
@@ -565,6 +565,14 @@ components:
           type : "array"
           items:
             type: "string"
+        bijbehorendeErfpachters:
```

Comment:
Ik had gedacht dat de erfpachter (ongeveer) dezelfde kenmerken had als een zakelijkgerechtigde. Omdat de erfpachter een zakelijkgerechtigde kan zijn van een andere koz, kan je niet zelf de url afleiden naar de zakelijkgerechtigde van de betreffende koz, zou in de bijbehorende erfpachters property bij _links geen templated urls bevatten. Dat is wat ik als toehoorder uit de meeting van gisteren concludeerde.

Reply:
klopt, maar wat bedoel je hiermee?

Ik dacht dat we geconcludeerd hadden dat we toch ook in de resource body (niet alleen in de HAL _links) een identificatie zouden opnemen. Aangezien dat niet met een enkele identificatie kan voor een zakelijk gerechtigde, doen we dat met een uri. Alternatief was geweest per zakelijk gerechtigde een object met zowel kadastraalonroerendezaakidentificatie als zakelijkgerechtigdeidentificatie op te nemen. 
Of helemaal niet in de body en alleen in de HAL links?

Hoe had jij dit logisch of makkelijker gevonden?

Reply:
Het voelt raar aan als je hebt over bijbehorendeErfpachters en het is een lijst van urls. Ik zou het dan gelijk getrokken hebben met de andere properties (bijbehorendeAppartementsrechtIdentificaties) en het bijbehorendeErfpachtersLinks of zoiets genoemd hebben, maar dat staat al in _links. Volgens mij is dan het niet nodig om het dubbel te doen en het ook als property op te nemen. Wat ik wel mis is dat het niet in _embedded voorkomt. Dat betekent dat als ik geïnteresseerd ben in de erfpachters dat ik altijd meerdere calls moet doen, één keer om de links op te halen en daarna elke zakelijkgerechtigde

Reply:
Mee eens. ik haal het gegeven weg uit de body van de resource. En voeg het toe aan _embedded. Property bijbehorendeErfpachters zit dan alleen nog in _links en _embedded.